tests/foreman/api/test_contentview.py

```diff
@@ -942,85 +822,6 @@ def initiate_testclass(
         module_cv.update(['repository'])
         request.cls.yumcv = module_cv.read()
 
-    @pytest.mark.tier2
-    def test_positive_add_rh(self):
-        """associate Red Hat content in a view
-
-        :id: f011a269-813d-4e82-afe8-f106b23cb03e
-
-        :expectedresults: RH Content assigned and present in a view
-
-        :CaseImportance: High
-        """
-        assert len(self.yumcv.repository) == 1
-        assert self.yumcv.repository[0].read().name == REPOS['rhst7']['name']
-
-    @pytest.mark.tier2
-    def test_positive_add_rh_custom_spin(self, target_sat):
```

Comment:
Is this test consider by mistake, I don't see any recommendation in sheet.

Reply:
just added it back.